### ISSUE TITLE
Invalid field type: "Item Count" must be unsigned integer when provided

### ISSUE THREAD

**Issue by MattApril** [stechstudio/bai2#12]
```
Fatal error: Uncaught STS\Bai2\Exceptions\MalformedInputException: Encountered issue trying to parse Account Identifier and Summary Status Field. Invalid field type: "Item Count" must be unsigned integer when provided. in /home/vagrant/code/billing-api/vendor/stechstudio/bai2/src/Records/TryableParserRecordTrait.php:24
Stack trace:
#0 /home/vagrant/code/billing-api/vendor/stechstudio/bai2/src/Records/AccountRecord.php(104): STS\Bai2\Records\AccountRecord->tryParser()
#1 /home/vagrant/code/billing-api/vendor/stechstudio/bai2/src/Records/AccountRecord.php(41): STS\Bai2\Records\AccountRecord->tryHeaderParser()
#2 /home/vagrant/code/billing-api/vendor/stechstudio/bai2/src/Records/GroupRecord.php(46): STS\Bai2\Records\AccountRecord->toArray()
#3 [internal function]: STS\Bai2\Records\GroupRecord->STS\Bai2\Records\{closure}()
#4 /home/vagrant/code/billing-api/vendor/stechstudio/bai2/src/Records/GroupRecord.php(47): array_map()
#5 /home/vagrant/code/billing-api/vendor/stechstudio/bai2/src/Records/FileRecord.php(42): STS\Bai2\Records\GroupRecord->toArray()
#6 [internal function]: STS\Bai2\Records\FileRecord->STS\Bai2\Records\{closure}()
#7 /home/vagrant/code/billing-api/vendor/stechstudio/bai2/src/Records/FileRecord.php(43): array_map()
#8 /home/vagrant/code/billing-api/vendor/stechstudio/bai2/bin/bai2json(33): STS\Bai2\Records\FileRecord->toArray()
#9 /home/vagrant/code/billing-api/vendor/bin/bai2json(117): include('...')
#10 {main}
  thrown in /home/vagrant/code/billing-api/vendor/stechstudio/bai2/src/Records/TryableParserRecordTrait.php on line 24
```

First 4 lines of the file:
```
01,000300032,123456789,170410,0616,1,80,,2/
02,123456789,000300032,1,170407,0616,CAD,2/
03,00002000001122334,CAD,010,+0004235036087,,,015,+0004639515887,,/
88,100,+0000410365913,+00011,,400,+0000005886113,+00001,/
```

This seems like a bug, because I am not providing the Item Count.

**Comment by MattApril** [comment]
I discovered that it was the leading + sign in front of the item count.

**Comment by bayprogrammer** [comment]
@MattApril thank you for the report. Is it the `+0004235036087` value which led to this error? Double-checking the spec and I see that it does not prohibit signed item counts, so I think this is indeed a bug. I saw your PRs, will take a look at those shortly.

**Comment by bayprogrammer** [comment]
Fixed by #14.
